src/ui/flutter_app/lib/screens/home_drawer.dart

```diff
@@ -271,7 +269,7 @@ class HomeDrawer extends StatelessWidget {
             0.0,
           ),
           child: Padding(
-            padding: const EdgeInsets.symmetric(horizontal: 8),
+            padding: const EdgeInsets.symmetric(vertical: 8),
```

Comment:
This change may have an effect on the style, I need to compile it out and check the effect.